From da76bd7120a6391657f7735503310f5145de3159 Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Fri, 27 Nov 2015 11:20:49 +0800
Subject: [PATCH 0703/5242] MLK-11915-07 ASoC: imx-audmux: fix missing break
 in switch

commit  549d6681841103b3abd2d860a4f299799ec33644 from
https://source.codeaurora.org/external/imx/linux-imx.git

fix missing break in switch. Reported by Coverity.

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
(cherry picked from commit 302516a1dfca5758a34cbed939c9976679febb0c)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/imx-audmux.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/fsl/imx-audmux.c b/sound/soc/fsl/imx-audmux.c
index 99124fd..f01a13e 100644
--- a/sound/soc/fsl/imx-audmux.c
+++ b/sound/soc/fsl/imx-audmux.c
@@ -343,6 +343,7 @@ static int imx_audmux_probe(struct platform_device *pdev)
 		break;
 	case IMX21_AUDMUX:
 		reg_max = 6;
+		break;
 	default:
 		dev_err(&pdev->dev, "unsupported version!\n");
 		return -EINVAL;
-- 
1.7.9.5

